resources/web/style/definition_hover.pcss

```diff
@@ -0,0 +1,36 @@
+#guide {
+  .Definition {
+    position: relative;
+
+    &:hover {
+      color: black;
+      transition: all 0.3s ease;
+      .Definition-defined {
+        z-index: 10000;
```

Comment:
How much of these styles are borrowed from elsewhere? Is there an opportunity to reuse them so they aren't specific to this element?